tests/bgv/verifier.mlir

```diff
@@ -0,0 +1,13 @@
+// RUN: heir-opt --verify-diagnostics --split-input-file %s
```

Comment:
Nit: `--split-input-file` is only necessary if you have multiple tests in the same file that must be parsed as if they were separate files (it's a weird convention), and the separator to mark when to split the file is `// -----`

Reply:
Applied!